certified-connectors/OpenLegacy IBM I (AS400)/apiDefinition.swagger.json

```diff
@@ -1,318 +1,477 @@
-                                "x-ms-visibility": "internal",
-                                "properties": {
-                                    "routingKey": {
-                                        "type": "string",
-                                        "description": "method name",
-                                        "title": "routingKey",
-                                        "x-ms-visibility": "internal"
-                                    }
-                                },
-                                "required": [
-                                    "routingKey"
-                                ]
-                            },
-                            "combinedRoute": {
-                                "type": "string",
-                                "description": "created by policy",
-                                "title": "combinedRoute",
-                                "x-ms-visibility": "internal"
-                            }
-                        },
-                        "required": [
-                            "id",
-                            "enrichment"
-                        ]
-                    },
-                    "description": "elements"
-                }
-            }
-        }
-    },
-    "parameters": {},
-    "responses": {},
-    "securityDefinitions": {
-        "api_key": {
-            "type": "apiKey",
-            "in": "header",
-            "name": "x-api-key"
+        {
+          "propertyName": "Categories",
+          "propertyValue": "Business Management; Data"
         }
-    },
-    "security": [],
-    "tags": []
+      ],
+  "consumes": [
+      "application/json"
+  ],
+  "produces": [
+      "application/json"
+  ],
+  "paths": {
+      "/api/v1/projects": {
+          "get": {
+              "summary": "Get All Projects",
+              "operationId": "GetAllProjects",
+              "x-ms-visibility": "internal",
+              "parameters": [
+                  {
+                      "name": "with_contracts",
+                      "type": "boolean",
+                      "in": "query",
+                      "default": true,
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines that Contract ID must be included in the response.",
+                      "description": "Determines that Contract ID must be included in the response.",
+                      "required": true
+                  },
+                  {
+                      "name": "module_type",
+                      "type": "string",
+                      "in": "query",
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines the BackendSolution of module by which projects must be filtered.",
+                      "description": "Determines the BackendSolution of module by which projects must be filtered.",
+                      "required": true
```

Comment:
@boiv The required parameter has been added. can you please confirm this change?

Reply:
Yes, this is a new parameter which is required